snyk_sync/utils.py

```diff
@@ -298,18 +302,26 @@ def ensure_dir(directory: Path) -> bool:
 
 def load_watchlist(cache_dir: Path) -> SnykWatchList:
     tmp_watchlist = SnykWatchList()
+    cache_data_errors = []
 
     if path.exists(f"{cache_dir}/data.json"):
         try:
             cache_data = jopen(f"{cache_dir}/data.json")
```

Comment:
Does cache_data = jopen(f"{cache_dir}/data.json") - handles the closing of the file?

Reply:
yes, it does.  that function uses "with" for the open file handle, when then with block exits the file is automatically closed.